common/lib/capa/capa/xqueue_interface.py

```diff
@@ -15,6 +15,8 @@
 
 # Wait time for response from Xqueue.
 XQUEUE_TIMEOUT = 35  # seconds
+CONNECT_TIMEOUT = 3.05
```

Comment:
Please mention in what units 3.05 and 10 is, and also you may want to give this a more appropriate name.
CONNECT_TIMEOUT -> CONNECTION_TIMEOUT.
I am ok with this also, just a suggestion.